Sources/RemoteNotification.swift

```diff
@@ -33,28 +33,39 @@ public struct RemoteNotification: CustomStringConvertible, Equatable {
     /// The identifier for the UIUserNotificationCategory associated with the push notification. Derived from userInfo["category"]
     public let categoryIdentifier: String?
     /// Custom fields on the remote notification.
-    public let userInfo: [String : AnyObject]
+    public let userInfo: [String : Any]
     /// The dictionary representation of the remote notification.
-    public let remoteNotificationDictionary: [String : AnyObject]
+    public let remoteNotificationDictionary: [String : Any]
```

Comment:
It's unfortunate that these `Any` types are here. If it were possible to create a protocol for expected types, that would be nice. I'm assuming this is just how the Cocoa API provides it.